src/Validation/ValidationConstraint.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+namespace WMDE\Fundraising\Frontend\Validation;
+
+/**
+ * Validate an instance property and add the source information to the ConstraintViolation
```

Comment:
The grammar here is off - this is an instruction, and not a description of what something does